src/main/java/org/threadly/concurrent/event/ListenerHelper.java

```diff
@@ -90,7 +89,7 @@ protected T makeProxyInstance(Class<? super T> listenerInterface) {
       if (listeners == null) {
         return Collections.emptyList();
       } else {
-        return Collections.unmodifiableList(new ArrayList<T>(listeners.keySet()));
+        return Collections.unmodifiableList(Pair.collectLeft(listeners));
```

Comment:
Do we really need collectLeft?  It seems like we are already guarding from null lefts on our add.  Seems like more work for no reason.